packages/kit/src/core/adapt/builder.js

```diff
@@ -216,3 +221,11 @@ async function compress_file(file, format = 'gz') {
 
 	await pipe(source, compress, destination);
 }
+
+/**
+ * @param {string[]} files
+ * @returns {string[]}
+ */
+export function filter_vite_files(files) {
```

Comment:
I think we don't need this now that we're passing the filter to `copy`

Reply:
whoops. Good catch